app/main.py

```diff
@@ -1 +1,41 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        appetite = 3
+        super().__init__(name=name, appetite=appetite, is_hungry=is_hungry)
+
+    def catch_mouse(self) -> None:
+        print("The hunt began!")
+
+
+class Dog(Animal):
+    def __init__(self, name: str, is_hungry: bool = True) -> None:
+        appetite = 7
+        super().__init__(name=name, appetite=appetite, is_hungry=is_hungry)
```

Comment:
same here, also you can simplify your code by adding appetite into annotations or in super().__init__  directly

Reply:
I would like to disagree here, as adding it into annotation will allow user to change it while creating a Cat (or a Dog) change appetite while creating an instance. And there is a well-known fact that all cats eat 3 food points at a time :)